Database_Restructure/DB_restructure_documentation.md

```diff
@@ -0,0 +1,129 @@
+
+### Assignments
+
+There has been a significant amount of discussion around how the assignments (pairings of mentors with mentees) will be represented in the database. Some ideas were:
+
+1. A separate collection for assignments
+2. Assignments as a field (list of profile_ids) in Mentor or Mentee collection
+
+These were based on the understanding that the assignments would be somewhat concrete, and once a mentor/mentee pairing was made, this pair would have regular meetings. However, due to the natural volatility of having volunteer mentors (also, the stakeholder has been clear on the fact that a dedicated assignment is asking too much of the mentors), it's unlikely that tracking explicit assignments is a pragmatic approach.
+
+The DS team has lightly discussed using meetings with positive feedback from mentors and mentees as an indication that the pair should be matched in the future (and might be implemented in a matching algorithm), but this is unnecessary for the MVP.
+
+### Mentee Progression
+
+Mentee progression was a way to track where a mentee was in the underdog program, which would be marked with one of several text values.
+
+This has been previously discussed, and it was determined that tracking a mentee's specific progression status within the program is unnecessary. A mentee's status in the program will be tracked with the simpler `is_active` boolean in the Mentee collection.
+
+### Feedback
+
+Feedback has been brought up in many meetings but exact details have not been decided. There's generally been two ideas:
+
+1. Feedback is provided by mentee (and optionally for mentors) after a meeting has concluded. This is feedback on how the meeting went, if the individual thought their pairing was a compatible match, etc. This is the only place for a mentor or mentee to provide feedback.
+2. Feedback can be provided outside of meetings. The purpose and nature of feedback outside of meetings is undecided.
+
+If feedback can only be provided after meetings, it makes sense to keep feedback in the meeting collection and delete the separate Feedback collection(s).
+If feedback can be provided outside of meetings, it should remain a separate collection.
+
+Nothing necessarily needs to be changed in the feedback collections for the MVP, but it should be decided if/how NLP and sentiment analysis will be applied to meeting feedback (the analyses currently only use the Feedback collection).
+
+## Updating this Documentation
+
+Underdog developers are expected to update the "Current State" and "Future Needs and Considerations" sections as progress is made.
+
+restructuring_research.xls is historical and should not be altered; only use this for reference.
+
+mongodb_schema_design.xls should be updated as changes are made to the design and the codebase, especially when a decision is reached regarding adding or altering a collection (Ticket, Feedback).
```

Comment:
WOW! I understand so much more about how everything is connected. Thank you for being explicit--I don't envy maintaining this file and the spreadsheet. There are a lot of ideas yet to be clarified.